cgi-bin/LJ/ConfCheck/General.pm

```diff
@@ -759,7 +758,6 @@ my %bools = (
              "OPENID_COMPAT" => "Support pre-1.0 OpenID specs as well as final spec.",
              "OPENID_STATELESS" => "Speak stateless OpenID.  Slower, but no local state needs to be kept.",
              "ONLY_USER_VHOSTS" => "Don't allow www.* journals at /users/ and /~ and /community/.  Only allow them on their own user virtual host domains.",
-             "USERPIC_MOGILEFS" => "Store userpics on MogileFS.",
```

Comment:
Yeah. Deleted as-is, but I'll probably use parts of it for BlobStore migration code, maybe.